src/main/java/com/parasoft/findings/sonar/sensor/CoverageSensor.java

```diff
@@ -124,11 +128,15 @@ public void uploadFileCoverageData(File report, SensorContext context) {
                             coverage.lineHits(lineNumber, hits);
                         }
                         coverage.save();
+                    } else {
+                        Logger.getLogger().warn(NLS.bind(Messages.FileInReportNotFound, filename));
                     }
                 }
             }
         } catch (DocumentException e) {
             Logger.getLogger().error(NLS.bind(Messages.FailedToLoadCoberturaReport, report.getName()), e);
+        } catch (Exception e) {
+            Logger.getLogger().error(e.getMessage(), e);
```

Comment:
The message should be changed to like Failed to upload cubertura report: pathToCuberturaReport.